examples/microprofile/idcs/src/main/resources/application.yaml

```diff
@@ -34,18 +34,27 @@ security:
     frontend-uri: "http://localhost:7987"
     proxy-host: "if you need proxy"
   providers:
-  - abac:
-  - oidc:
-      client-id: "${ALIAS=security.properties.idcs-client-id}"
-      client-secret: "${ALIAS=security.properties.idcs-client-secret}"
-      identity-uri: "${ALIAS=security.properties.idcs-uri}"
-      # A prefix used for custom scopes
-      scope-audience: "http://localhost:7987/test-application"
-      proxy-host: "${ALIAS=security.properties.proxy-host}"
-      frontend-uri: "${ALIAS=security.properties.frontend-uri}"
-      # Retrieve roles
-      idcs-roles: true
+    - abac:
```

Comment:
does the empty abac key do something? If so, please add a comment. Otherwise probably remove it

Reply:
yes it does - it adds the `ABAC` provider to the list of security providers used

Reply:
I have added a comment to each of these used with OIDC/IDCS in examples